src/routes/routes.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const fsRoute = require('./fs')
+const geocodeRoute = require('./geocode')
+const forecastRoute = require('./forecast')
+
+const app = express()
+const port = 3000
+
+app.use('/fs', express.raw({ type: "application/json" }), fsRoute)
```

Comment:
You don't need to copy paste which parser to use.

use one line instead
`app.use(bodyParser.json({ limit: '64kb' })); // for parsing application/json`